xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -663,10 +719,21 @@ The property name is escaped before writing.
         <Parameter Name="value" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="propertyName">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="propertyName">The JSON encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
+        <param name="value">The value to be written as a JSON literal <b>true</b> or <b>false</b> as part of the name/value pair.</param>
+        <summary>Writes the pre-encoded property name and <see cref="T:System.Boolean" /> value (as a JSON literal <b>true</b> or <b>false</b>) as part of a name/value pair of a JSON object.</summary>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+
+## Remarks
+
+The property name should already be escaped when the instance of <xref:System.Text.Json.JsonEncodedText> was created.
+
+            ]]>
+          </format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">Validation is enabled, and this method would result in writing invalid JSON.</exception>
       </Docs>
     </Member>
     <Member MemberName="WriteBooleanValue">
```

Comment:
Just like `WriteNullValue`, we should fix this one too:

```diff
- The value to be written as a JSON literal "true" or "false" as an element of a JSON array.
+ The value to be written as a JSON literal <b>true</b> or <b>false</b> as an element of a JSON array.
```

Reply:
I fixed all the JSON literals I found in that file.